conformance_tests/tools/metrics/src/test_metric.cpp

```diff
@@ -438,49 +438,42 @@ class zetMetricQueryTest : public zetMetricGroupTest {
   std::vector<std::string> groupNameList;
   zet_metric_group_handle_t matchedGroupHandle{};
   std::string groupName;
-
-  void zetMetricQueryInitialize() {
+  void SetUp() override {
     groupNameList = lzt::get_metric_group_name_list(
         device, ZET_METRIC_GROUP_SAMPLING_TYPE_FLAG_EVENT_BASED, false);
-    ASSERT_GT(groupNameList.size(), 0u)
-        << "metric group name list cannot be empty";
+    ASSERT_GT(groupNameList.size(), 0u) << "No query metric groups found";
```

Comment:
did you confirm the test can fail if the setup() fails ?

Reply:
Yes, the test will fail if the SetUp() fails, Thank you.